hap.coffee

```diff
@@ -112,6 +120,20 @@ module.exports = (env) =>
         env.logger.debug("unsupported device type: " + device.constructor.name)
         return null
 
+    createAccessoriesFromTemplate: (device) =>
```

Comment:
Would be great if ButtonAccessory could handle multiple buttons itself, but this needs some rework at adding the accessory to the bridge, i guess.